Journaley/Forms/MainForm.cs

```diff
@@ -475,7 +475,8 @@ private Markdown Markdown
                 if (this.markdown == null)
                 {
                     this.markdown = new Markdown();
-                    ((MarkdownOptions)this.markdown.Options).AutoNewLines = true;
+                    this.markdown.ExtraMode = true;
+                    this.markdown.SafeMode = false;

```

Comment:
From the [website](http://www.toptensoftware.com/markdowndeep/api):

>bool SafeMode
>Set to true to only allow whitelisted safe html tags
>
>bool ExtraMode
>Set to true to enable ExtraMode, which enables the same set of features as implemented by PHP Markdown Extra.

Extramode is needed, becuase most of the implementations in Day One uses Markdown Extra stuff like tables and footnotes